### ISSUE TITLE
Incompatible with ModernFix's Dynamic Resources

### ISSUE THREAD

**Issue by luciobortoletto** [MehVahdJukaar/polytone#86]
When using Polytone with Modernfix and Dynamic Resources enabled, blocks and items have a missing texture.  
  
Minecraft version: 1.21.1  
  
Fabric loader version: 0.16.3  
  
Installed mods:
- Fabric API 0.103.0+1.21.1
- Mod Menu 11.0.2
- ModernFix 5.19.3+mc1.21.1
- Polytone 1.21-2.4.7  
  
Logs: https://mclo.gs/QNKRVbY  
  
Screenshot: 
![2024-08-30_22 00 13](https://github.com/user-attachments/assets/da2e161b-3eb6-4f5a-8d11-51acaabddb9b)

**Comment by ItsPasi** [comment]
Just wanted to make the same bug report. The break happened with version 2.4.2 of Polytone, before it worked just fine.

**Comment by MehVahdJukaar** [comment]
that option is off by default for a reason 💀 
I'll see what i can do but i cant guarantee

**Comment by ItsPasi** [comment]
> that option is off by default for a reason 💀 I'll see what i can do but i cant guarantee

Yeah it's not too important to fix but since it worked in the past I thought there might be a straightforward fix to it.

**Comment by ItsPasi** [comment]
Little update, it seems this issue has been fixed with version 2.4.9 but it introduced a new issue where if you join a world the first frame when joining is just completely white. Loading up the game for the first time when joining still is laggy makes the screen white for up to a few seconds.
